src/dir_contents.ml

```diff
@@ -130,10 +182,9 @@ end = struct
           (list_modules missing_intf_only)
     end;
     if missing_modules <> [] then begin
-      let module_name = Module.name (List.hd missing_modules) in
-      let (loc, _) =
-        Module.Name.Map.find modules_without_implementation_locs module_name
-        |> Option.value_exn
+      let (loc, module_name) =
+        let (loc, module_) = List.hd missing_modules in
```

Comment:
This was in the original code, but I believe it'd be cleaner to refactor the `<> []` + `List.hd` into pattern matching.

Reply:
Will do that